docs/readthedocs/source/doc/Friesian/QuickStart/Train_wnd.md

```diff
@@ -0,0 +1,269 @@
+# Feature Engineering for W&D (Wide and Deep Learning) using Friesian
+With Friesian FeatureTable APIs, data scientists and machine learning engineers are able to quickly process datasets of all sizes to generate features for recommender models. This tutorial showcases how to extract features for [Wide and Deep Learning](https://arxiv.org/abs/1606.07792) (W&D) using [movielens dataset](http://files.grouplens.org/datasets/movielens/).
+
+## **1. Key Concepts**
+A **FeatureTable** provides distributed table for processing recsys feature, common tabular operations, as well as high-level recsys feature transformation. e.g.
+- Categorical Data Encoding, FeatureTable can encode categorical data into integers, one hot encodings, target encodes.
+- Cross-product Feature Transformations, FeatureTable crosses categorical columns and hash into certain buckets to generate more features.
+- Negative Sampling, FeatureTable selects negative examples randomly from the user’s non-interactive product set.
+
+**Wide and Deep Learning** is a noval recommender model proposed by Google. Specifically, A W&D model is jointly trained wide linear models and deep neural networks to combine the benefits of memorization and generalization for recommender systems.
```

Comment:
"A W&D model is jointly trained wide linear models and deep neural networks" - this does not sounds right